pkg/runner/generator.go

```diff
@@ -0,0 +1,103 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package runner
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/opensbom-generator/parsers/meta"
+	"github.com/opensbom-generator/parsers/plugin"
+
+	"github.com/spdx/spdx-sbom-generator/pkg/runner/options"
```

Comment:
I think this should be "github.com/open-sbom-generator/spdx-sbom-generator..."

Reply:
Eeee well more or less. In an ideal world yes but the module is still named under the SPDX org:

https://github.com/opensbom-generator/spdx-sbom-generator/blob/f702b3082f815e461b37a260c2adc14dfa091a2f/go.mod#L1-L8